packages/react/src/types.ts

```diff
@@ -26,11 +25,25 @@ declare global {
 
 export type IsomorphicClerkOptions = Without<ClerkOptions, 'isSatellite'> & {
   Clerk?: ClerkProp;
+  /**
+   * Define the URL that `@clerk/clerk-js` should be hot-loaded from
+   */
   clerkJSUrl?: string;
+  /**
+   * If your web application only uses Control components, you can set this value to `'headless'` and load a minimal ClerkJS bundle for optimal page performance.
+   */
   clerkJSVariant?: 'headless' | '';
+  /**
+   * Define the npm version for `@clerk/clerk-js`
+   */
   clerkJSVersion?: string;
-  sdkMetadata?: SDKMetadata;
+  /**
+   * The Clerk publishable key for your instance. This can be found in your Clerk Dashboard on the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page.
+   */
   publishableKey: string;
+  /**
+   * This nonce value will be passed through to the `@clerk/clerk-js` script tag. You can use this to implement [strict-dynamic CSP](https://clerk.com/docs/security/clerk-csp#implementing-a-strict-dynamic-csp).
```

Comment:
💭 We could also place those disclaimers on a `@note` tag, as I think it brings more attention when reading it on a separate line. 

![CleanShot 2024-09-04 at 07 26 48](https://github.com/user-attachments/assets/345c1b13-98b6-4137-9093-c3f7740287d6)